visidata/vd.py

```diff
@@ -1471,6 +1471,9 @@ def read_bytes(self):
     def is_dir(self):
         return os.path.isdir(self.resolve())
 
+    def exists(self):
+        return os.path.exists(self.resolve())
```

Comment:
Yes, this is good enough (focus in VisiData and this support class is simplicity/straightforwardness).  Thanks for getting the example to work, @zormit!